jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/utils/ClickAwayListenerTests.java

```diff
@@ -0,0 +1,71 @@
+package io.github.epam.material.tests.utils;
+
+import io.github.epam.TestsInit;
+import javafx.util.Pair;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+
+import static io.github.com.StaticSite.utilsClickAwayListenerDefaultPage;
+import static io.github.com.StaticSite.clickAwayListenerFrame;
+import static io.github.com.StaticSite.utilsClickAwayListenerPortalExamplePage;
+
+public class ClickAwayListenerTests extends TestsInit {
+
+    private ArrayList<Pair<Integer, Integer>> points;
+
+    @Test
+    public void defaultClickAwayListenerTest() {
+        utilsClickAwayListenerDefaultPage.open();
+        textFieldTest();
+    }
+
+    @Test
+    public void portalExampleClickAwayListenerTest() {
+        utilsClickAwayListenerPortalExamplePage.open();
+        textFieldTest();
+    }
+
+    private void textFieldTest() {
+        clickAwayListenerFrame.openMenuButton.click();
+        clickAwayListenerFrame.text.is().displayed();
+        //initializing all border points of a text field
+        initPoints();
+        //hiding text field by clicking on 'open menu dropdown' button
+        clickAwayListenerFrame.openMenuButton.click();
+        clickAwayListenerFrame.text.is().hidden();
+        //checking all border points of a text field
+        for (Pair<Integer, Integer> point : points) {
+            clickAwayListenerFrame.openMenuButton.click();
+            clickAwayListenerFrame.text.is().displayed();
+            clickAwayListenerFrame.text.click(point.getKey(), point.getValue());
```

Comment:
need to apply point.key and point.value instead of methods

Reply:
done